app/utils/getCacheAnalysis.test.ts

```diff
@@ -91,3 +359,305 @@ describe('getTimeToLive', () => {
     expect(getTimeToLive(age, date, expiresAt, maxAge, now)).toBeUndefined()
   })
 })
+
+// Test helper functions for internal functions since they're not exported
+// We'll test them through the main functions where possible, but these are edge cases
+
+describe('internal helper functions', () => {
```

Comment:
as mentioned elsewhere, don't do this... just export and add missing unit tests in the proper new test files as needed